cddl/examples/comid-series.diag

```diff
@@ -24,13 +24,12 @@
           [         
             { / *** measurement-map *** /
               / mval / 1 : / measurement-values-map / {
-                / ver / 0 : {
-                  / version / 0 : "1.0.0",
-                  / version-scheme / 1 : 16384 / semver /
+                / comid.flags / 3 : { 
+                  / configured / 0 : true
                 }
               },
               / authorized-by / 2 : [
-                / tagged-pkix-base64-key-type / 554("base64_key_X")
+                / tagged-pkix-base64-key-type / 554("base64_key_ACME_signer")
```

Comment:
Can we make this a syntactically valid 6.554 value like in the referenced documentation?

-----BEGIN PUBLIC KEY-----\nMHYwEAYHKoZIzj0CAQYFK4EEACIDYgAEn1LlwLN/KBYQRVH6HfIMTzfEqJOVztLe\nkLchp2hi78cCaMY81FBlYs8J9l7krc+M4aBeCGYFjba+hiXttJWPL7ydlE+5UG4U\nNkn3Eos8EiZByi9DVsyfy9eejh+8AXgp\n-----END PUBLIC KEY-----

Reply:
Issue #327

Reply:
> Issue #327

Given issue #327 addresses the concern. It doesn't need to be addressed in this PR.